content/docs/1.1.0/advanced-resources/deploy/revision_counter.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Revision Counter
+weight: 2
+---
+
+Revision Counter is a strong mechanism that Longhorn keeps tracking each replicas' updates. During replica creation, Longhorn will create a 'revision.counter' file, it's initial counter is 0. And for every write to the replica, the counter in 'revision.counter' file will be increased by 1. And Longhorn Engine will use these counters to make sure all replcias are consistent during start time. Also for salvage recovering, will use these counters to decide which replica has the latest update.
+
+Disable Revision Counter is an option to boost Longhorn performance which doesn't keep tracking every write on replicas. During Longhorn Engine start time, it will skip checking the revision counter for all replicas. But Longhorn still support auto-salvage through the replica's head file stat. By default the revision counter is enabled.
```

Comment:
Need to link to how Longhorn detect and choose the good replica if all the replicas are disconnected. Otherwise without a replacement, it would be dangerous to do so.

Reply:
Updated this part, please take a look again. Thanks.